### PR TITLE
[SwiftSyntaxBuilder] Add protocol conformance map

### PR DESCRIPTION
**Pull request by kimdv**
Added a map to help add protocols to SwiftSyntax Buildables

https://github.com/apple/swift-syntax/pull/320

### CODE REVIEW

File: utils/gyb_syntax_support/protocolsMap.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+PROTOCOL_CONFORMANCES_MAP = {
```
Comment (kimdv) [inline utils/gyb_syntax_support/protocolsMap.py:1]: Not sure if this names is a good fitting as it contains structs and protocols with their conformances

**Comment by ktoso** [comment]
@swift-ci please smoke test

**Review by ahoppen — APPROVED** [review]

**Comment by kimdv** [comment]
@ahoppen the MacOS build fails with 
```
20:37:14       #line: (FloatLiteralExpr(floatingDigits: .floatingLiteral(String(123.321))), "␣123.321"),
20:37:14 /Users/buildnode/jenkins/workspace/swift-PR-macos-smoke-test/branch-main/swift-syntax/Tests/SwiftSyntaxBuilderTest/FloatLiteralTests.swift:11:72: error: missing argument label 'describing:' in call
20:37:14       #line: (FloatLiteralExpr(floatingDigits: .floatingLiteral(String(-123.321))), "␣-123.321"),
```

When running locally it works. 
Can you help me out or point me in direction how to solve it?

**Comment by ahoppen** [comment]
I’m on vacation next week and will look at the failure when I get back. I would start off by checking if you also see the issue locally when you compile SwiftSyntax with the latest open source toolchain from [swift.org](https://swift.org/download/).

**Pull request merged**